### ISSUE TITLE
Question: Plans for a ReportViewer

### ISSUE THREAD

**Issue by shaggygi** [microsoft/microsoft-ui-xaml#1156]
Just wondering if there are plans to offer a ReportViewer control?  This is mainly a question to make sure this wasn't a duplicate issue.  If not, details can be modified with more information.  Thx

**Comment by jevansaks** [comment]
We don't have plans for such a thing. The Windows Community Toolkit has the DataGrid control now and we're looking at adding support for Flex and CSS Grid-style layouts which would enable table layouts like this very easily. Would either of those work for you?

**Comment by shaggygi** [comment]
I don't think so.  I'm thinking of a little more support for reports like using RDL files and printing as we use Microsoft.ReportViewer.* packages today.

**Comment by mrlacey** [comment]
@shaggygi Not what you're directly asking, but in that it might help unblock you, Syncfusion have such a control https://www.syncfusion.com/reporting-tool/uwp/report-viewer.
It's not free but it's the only one I'm aware of.

**Comment by shaggygi** [comment]
@mrlacey  Thanks for the link.  I've seen similar with other 3rd-party vendors, as well.  This issue was seeing if something similar would be provided in core controls/APIs.  Also did a similar ping related to [diagram control](https://github.com/microsoft/microsoft-ui-xaml/issues/730#issuecomment-493670179).
